tidb-cloud/serverless-tier-limitations.md

```diff
@@ -37,10 +37,6 @@ We are constantly filling in the feature gaps between Serverless Tier and Dedica
 - Only [Standard Connection](/tidb-cloud/connect-via-standard-connection.md) can be used. You cannot use [Private Endpoint](/tidb-cloud/set-up-private-endpoint-connections.md) or [VPC Peering](/tidb-cloud/set-up-vpc-peering-connections.md) to connect to Serverless Tier clusters. 
 - No "IP Access List" support.
 
-## Backup and Restore
-
-- [Backup and Restore](/tidb-cloud/backup-and-restore.md) are not supported for Serverless Tier currently.
```

Comment:
It seems that Serverless Tier doesn't support **manual backup**. Can you add this as a limitation?

Reply:
Sure. Just double check do you mean add manual backup and pitr as limitations in serverless-tier-limitations.md?